### ISSUE TITLE
Calendar Translation

### ISSUE THREAD

**Issue by applepipe** [frappe/translator#91]
Hello,

I'm translating to pt-BR and using in my company to test the translation and correcting some confusing terms directly in the portal, but I noted that calendar doesnt respect the language set, the default is always english aparently.

Looking in the forum I found this topic: https://discuss.erpnext.com/t/translation-for-the-calendar/7818

In a brief research I've found this topic in the documentation of fullcalendar: https://fullcalendar.io/docs/text/locale/

Basically it has support for my language, tested here https://fullcalendar.io/js/fullcalendar-3.0.0/demos/locales.html

I cant find where to edit my calendar language, the documentation says "locale" folder but I think this may be different in erpnext.

I think that calendar language should consider the system language, or at least time zone setting.

**Comment by fmorato** [comment]
The app seems broken from an user perspective if parts of the software are not translated.
